static/src/js/containers/TimelineContainer/TimelineContainer.js

```diff
@@ -71,18 +73,25 @@ export const TimelineContainer = (props) => {
   if (collectionsToRender.length === 0) return null
 
   return (
-    <Timeline
-      browser={browser}
-      collectionMetadata={collectionMetadata}
-      pathname={pathname}
-      showOverrideModal={isProjectPage}
-      temporalSearch={temporalSearch}
-      timeline={timeline}
-      onChangeQuery={onChangeQuery}
-      onChangeTimelineQuery={onChangeTimelineQuery}
-      onToggleOverrideTemporalModal={onToggleOverrideTemporalModal}
-      onMetricsTimeline={onMetricsTimeline}
-    />
+    <Suspense fallback={(
+      <div className="timeline-loading">
```

Comment:
Given the UI and 'data' that this is loading, we think it's safe to remove any loading indicators for this PR. So, please remove the fallback attribute here and all corresponding code.

Reply:
Sounds good. I have replaced it with a null value since the fallback property can't be omitted from the Suspense component.